spec/views/admin/index.html.erb_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'admin/index' do
+  let(:blacklight_config) { CatalogController.new.blacklight_config }
+  let(:ability) { Ability.new(user) }
+  before do
+    allow(view).to receive(:blacklight_config).and_return(blacklight_config)
+    allow(view).to receive(:has_user_authentication_provider?).and_return(true)
+    allow(view).to receive(:current_user).and_return(user) if user
+    allow(controller).to receive(:current_ability).and_return(ability)
+    render
+  end
+  context "when not logged in" do
+    let(:user) {nil}
+    it "should display an insufficient permissions error" do
+      expect(rendered).to_not have_content "Admin Panel"
+      expect(rendered).to have_content "You do not have sufficient permissions to access this page."
```

Comment:
So you should check that it has whatever links this page needs to have here. Then there should be a controller spec that makes sure it throws the right error if you're unauthorized and redirects.